### PR TITLE
[VEN-2680]: deployment of oracle on Zksync sepolia

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -900,6 +916,38 @@ export const assets: Assets = {
...
+      address: "0xeF891B3FA37FfD83Ce8cC7b682E4CADBD8fFc6F0",
+      oracle: "chainlink",
+    },
+    {
+      token: "WETH",
+      address: "0x53F7e72C7ac55b44c7cd73cC13D4EF4b121678e6",
+      oracle: "chainlink",
+    },
+    {
+      token: "USDC",
```
Comment (chechu) [inline helpers/deploymentConfig.ts:931]: Maybe we should use `USDC.e` instead, to replicate the symbol of the underlying token on mainnet

**Review by chechu — COMMENTED** [review]
Could we rebase this PR on top of https://github.com/VenusProtocol/oracle/pull/205? #205 should be merged before this PR, and it includes some changes that could generate a conflict with the changes included in this PR

File: helpers/deploymentConfig.ts (hunk 1 above)
Comment (GitGuru7) [reply helpers/deploymentConfig.ts]: [b49cb79](https://github.com/VenusProtocol/oracle/pull/209/commits/b49cb79556d1b1d753ca71ffcee884ece5051d4c)

File: deploy/1-deploy-oracles.ts [hunk 2]
```diff
@@ -29,18 +29,27 @@ const func: DeployFunction = async function ({ getNamedAccounts, deployments, ne
...
   await deploy("BoundValidator", {
     from: deployer,
     log: true,
     deterministicDeployment: false,
+    skipIfAlreadyDeployed: true,
     args: [],
     proxy: {
       owner: proxyOwnerAddress,
-      proxyContract: "OptimizedTransparentProxy",
+      proxyContract: "OptimizedTransparentUpgradeableProxy",
```
Comment (chechu) [inline deploy/1-deploy-oracles.ts:44]: Any mandatory reason to make this change?

Comment (GitGuru7) [reply deploy/1-deploy-oracles.ts:44]: `OptimizedTransparentProxy` is basically a flag. Hardhat-deploy plugin checks this and adds the `OptimizedTransparentUpgradeableProxy` and the `DefaultProxyAdmin` fields in the places I have added them in. I only have brought it one level up from the plugin in the node modules. The reason for this is that the contracts `OptimizedTransparentUpgradeableProxy` and `DefaultProxyAdmin` that the hardhat-deploy plugin fetches from the artifact is not zk compatible causing the deployments to fail. So we bought it one level up to our repo, added them to compile using zksync compiler.

**Review by chechu — APPROVED** [review]

**Pull request merged**